src/components/formComponentTemplate.tsx

```diff
@@ -105,6 +221,53 @@ export const formComponentTemplate = (
         ),
         validate: validate.positiveInt,
       };
+    case "liquidity_pool_id":
+      return {
+        render: (templ: TemplateRenderProps) => (
+          <TextPicker
+            key={id}
+            id={id}
+            label="Liquidity Pool ID"
+            labelSuffix={!templ.isRequired ? "optional" : undefined}
+            placeholder="Ex: 67260c4c1807b262ff851b0a3fe141194936bb0215b2f77447f1df11998eabb9"
+            value={templ.value || ""}
+            error={templ.error}
+            onChange={templ.onChange}
+          />
+        ),
+        validate: null,
+      };
+    case "offer_id":
+      return {
+        render: (templ: TemplateRenderProps) => (
+          <TextPicker
+            key={id}
+            id={id}
+            label="Offer ID"
+            labelSuffix={!templ.isRequired ? "optional" : undefined}
+            value={templ.value || ""}
+            error={templ.error}
+            onChange={templ.onChange}
+          />
+        ),
+        validate: null,
+      };
+    case "operation":
+      return {
+        render: (templ: TemplateRenderProps) => (
+          <PositiveIntPicker
+            key={id}
+            id={id}
+            label="Operation ID"
+            placeholder="Ex: 55834578945"
+            labelSuffix={!templ.isRequired ? "optional" : undefined}
+            value={templ.value || ""}
+            error={templ.error}
+            onChange={templ.onChange}
+          />
+        ),
+        validate: validate.positiveInt,
```

Comment:
I get an error `value.charAt is not a function` from `validate/methods/positiveInt.ts` when I go to a different page after clicking validating. It looks like on a new render, the value becomes number instead of string and throws an error in `validate/methods/positiveInt.ts`.

Steps to reproduce:
1. Go to `Single Ledger` page and submit the form with a ledger sequence number
2. Go to a different page
3. You will get an error `value.charAt is not a function` from `validate/methods/positiveInt.ts` and its value is getting `number` instead of `string`

Reply:
Thanks for catching this, @jeesunikim ! :raised_hands: I'll fix it.